Source/WebCore/WebCorePrefix.h

```diff
@@ -35,9 +35,9 @@
 
 #if defined(__APPLE__)
 #ifdef __cplusplus
-#define NULL __null
+#include <cstddef>
 #else
-#define NULL ((void *)0)
+#include <stddef.h>
```

Comment:
What do we need these for? It's a lot to pull into the prefix header.

Reply:
the error that I am trying to fix here is that this definition is confliciting with stddef.h so to permanently fix this I am just letting the header deal with it. It's anyways included further down in another file with which it is conflicting with.

Reply:
Interesting… why is this an issue now? Is it because we've turned on clang modules? (This change seems fine, but just curious)

Reply:
AFAIK...Unified Sources do expose issues like this whilst adding new files/reordering files.